### PR TITLE
Fixes #142 - Updated first and last name fields validation

### LINKED ISSUE #142: Change validation on firstname and lastname fields using HMRC pattern

### PR DESCRIPTION
**Pull request by s-barrah**
Fixes #142 - Updated first and last name fields validation

### CODE REVIEW

File: src/pages/GiftAidForm/defaultGiftaidFields.json [hunk 1]
```diff
@@ -25,13 +25,17 @@
     "type": "text",
     "name": "firstname",
     "label": "First name",
-    "required": true
+    "required": true,
+    "pattern": "^[A-Za-z][A-Za-z\\'\\-]*$",
+    "invalidErrorText": "This field only accepts alphabets and ' - "
```
Comment (Heleen-cr) [inline src/pages/GiftAidForm/defaultGiftaidFields.json]: perhaps alphabetical characters instead of alphabets

Comment (Heleen-cr) [reply src/pages/GiftAidForm/defaultGiftaidFields.json]: alphabetic charactes.

Please ask a native ;)

Comment (AndyEPhipps) [reply src/pages/GiftAidForm/defaultGiftaidFields.json]: "Alphabetic characters" 👍

Comment (AndyEPhipps) [reply src/pages/GiftAidForm/defaultGiftaidFields.json]: Didn't see say were gonna change this to also accept umlauts, accents etc too?

@CarolineRennie ?

Comment (KrupaPammi) [reply src/pages/GiftAidForm/defaultGiftaidFields.json]: This is the pattern type Corin mentioned 
First name: Firstname `[A-Za-z][A-Za-z'\-]*`
Last Name: [A-Za-z0-9 ,\.\(\)/&\-']+

**Pull request merged**